clustered/client/src/main/java/org/ehcache/clustered/client/internal/EhcacheClientEntity.java

```diff
@@ -327,7 +327,7 @@ private EhcacheEntityResponse invokeInternal(TimeoutDuration timeLimit, EhcacheE
    */
   public static final class Timeouts {
 
-    public static final TimeoutDuration DEFAULT_READ_OPERATION_TIMEOUT = TimeoutDuration.of(5, TimeUnit.SECONDS);
+    public static final TimeoutDuration DEFAULT_READ_OPERATION_TIMEOUT = TimeoutDuration.of(20, TimeUnit.SECONDS);
```

Comment:
Because the replication tests fail due to get time out. Since the fail-over takes more than 15 secs on my machine atleast.